lib/util/got/index.ts

```diff
@@ -1,14 +1,15 @@
 import got from 'got';
-import auth from './auth';
-import { mergeInstances } from './util';
+import { create, mergeInstances } from './util';
 
 export * from './common';
 
-/*
- * This is the default got instance for Renovate.
- *  - Cache all GET requests for the lifetime of the repo
- *
- */
-export const api = mergeInstances(got, auth);
+const dummy = create({
+  options: {},
+  handler: (options, next) => {
+    return next(options);
+  },
+});
+
+export const api = mergeInstances(got, dummy);
```

Comment:
What happens when we simply directly use got? Again lots of failing tests?

Reply:
Yep. Actually lots of tests mock util/got so they immediately fail. I tried changing them over but it was dozens of test suites failures either way